.travis.yml

```diff
@@ -0,0 +1,42 @@
+# references:
+# * http://www.objc.io/issue-6/travis-ci.html
+# * https://github.com/supermarin/xcpretty#usage
+
+language: objective-c
+
+# cache: cocoapods
+podfile: Example/Podfile
+
+before_install:
+  - brew install swiftlint
+  - gem install cocoapods --pre
+
+osx_image:
+  - xcode7.3
+  - xcode8
+# env:
+#   matrix:
+#     - TASK=build
+#     - TASK=lintpod
+
+matrix:
+    include:
+        - os: osx
+          osx_image: xcode8
+          env:
+            - TASK=build
+              DESTINATION="platform=iOS Simulator,name=iPhone 6,OS=10.0"
+        - os: osx
+          osx_image: xcode8
+          env: TASK=lintpod
+        # - os: osx
+        #   osx_image: xcode7.3
+        #   env:
+        #     - TASK=build
+        #       DESTINATION="OS=9.3,name=iPhone 6,platform=iOS Simulator"
+        - os: osx
+          osx_image: xcode7.3
```

Comment:
This is a good point to bring up. The Swift 3 support is being tested on the `xcode8` osx_image (both the build and linting the CocoaPods spec) . 

I had been assuming the Swift 2 compatibility could be fully tested by using Xcode 7. However, that's only testing Swift 2.2 support. To actually test Swift 2.3 support, it has to build on Xcode 8 with `SWIFT_VERSION` set to `2.3`. This can probably be handled with another variable in `.travis.yml` to pivot off of and use the value to be used as the `SWIFT_VERSION`.